### PR TITLE
[THORN-2357] update to Thorntail 2.4.0.Final

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

**Comment by Ladicek** [comment]
retest this please

File: src/licenses/licenses.xml [hunk 1]
```diff
@@ -649,7 +649,7 @@
       </licenses>
     </dependency>
     <dependency>
-      <groupId>io.openshift.boosters</groupId>
+      <groupId>io.openshift.booster</groupId>
```
Comment (sberyozkin) [inline src/licenses/licenses.xml]: What is this groupId about ? (As we are dropping the 'booster')

File: src/main/java/io/thorntail/example/GreetingResource.java [hunk 2]
```diff
@@ -13,30 +13,27 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package io.openshift.boosters;
+package io.thorntail.example;
 
-import java.net.InetAddress;
+import org.jboss.as.controller.client.ModelControllerClient;
```
Comment (sberyozkin) [inline src/main/java/io/thorntail/example/GreetingResource.java]: May be an import order issue ?

File: src/licenses/licenses.xml (hunk 1 above)
Comment (Ladicek) [reply src/licenses/licenses.xml]: Licenses will be regenerated during booster release. I just automatically replaced `io.openshift.boosters` by `io.openshift.booster` before renaming entirely.

File: src/main/java/io/thorntail/example/GreetingResource.java (hunk 2 above)
Comment (Ladicek) [reply src/main/java/io/thorntail/example/GreetingResource.java]: I did reformat all imports, it's IntelliJ style now :-) `java.net.InetAddress` is no longer used.

Comment (sberyozkin) [reply src/main/java/io/thorntail/example/GreetingResource.java]: I guess it is not a problem `javax.` is at the bottom

**Comment by Ladicek** [comment]
retest this please

**Pull request merged**